--- debian/control	2013-12-15 20:07:47.175185044 +0100
+++ debian/control	2013-12-15 20:08:03.247184863 +0100
@@ -2,7 +2,7 @@
 Section: tex
 Priority: optional
 Maintainer: Stefan Löffler <st.loeffler@gmail.com>
-Build-Depends: debhelper (>= 6), pkg-config, cmake (>= 2.8.6), libqt4-dev (>= 4.6), zlib1g-dev, libhunspell-dev, libpoppler-qt4-dev, liblua5.1-0-dev
+Build-Depends: debhelper (>= 6), pkg-config, cmake (>= 2.8.6), libqt4-dev (>= 4.6), zlib1g-dev, libhunspell-dev, libpoppler-qt4-dev, liblua5.2-dev
 Standards-Version: 3.8.0
 Homepage: https://github.com/TeXworks/
 

